s3sync.go

```diff
@@ -65,6 +67,13 @@ type fileOp struct {
 	op operation
 }
 
+type syncStatistics struct {
+	Bytes              int64
+	Files              int64
+	Time               int64
```

Comment:
- I'm not very sure what `Time` mean. It seems be an aggregated duration of up/download in milliseconds, but as multiple up/downloads are done in parallel, there are duplication
- Unit of duration is better to be `time.Duration`

Reply:
Yeah, `Time` here is time used while uploading/downloading per file. Maybe this should be calculated at the sync level (how long it took for the sync to complete, not at the file level as it is right now. WDYT? Or perhaps best to leave it out?

My intention was to capture how long it took per file, assuming serial upload/download, but with parallel in place it makes no sense as it is. I wonder if it makes sense to capture per sync operation..

Reply:
Changed time field name to `SyncTime` and moved the logic to capture the time it takes to complete the sync, rather than the transfer of individual files. At this point I'm not sure if it's still worth keeping this field in place, since it's easy to wrap the `Sync` call to capture the time it takes to complete:
```go
startTime:=time.Now()
err:=m.Sync(...)
totalTime:=(time.Now().UnixNano() - startTime.UnixNano())/int64(time.Millisecond)
```
 I leave it up to you, if you don't think it's worth it I'll remove it.

Reply:
> At this point I'm not sure if it's still worth keeping this field in place, since it's easy to wrap the `Sync` call to capture the time it takes to complete:

It seems good to be just removed as wrapping `Sync` to measure the duration is easy enough.

Reply:
Done.